widgetry/src/style/buttons.rs

```diff
@@ -118,8 +118,16 @@ pub trait StyledButtons<'a> {
         self.btn_secondary_dark_dropdown().label_text(text)
     }
 
+    /// An "X" button to close the current state. Bound to the escape key and aligned to the right,
```

Comment:
Mentioning right alignment here isn't accurate. But also, why expose the builder for this case? Currently, everything just asks for the widget. If somebody wanted to customize the close button somehow, at what point is it better for them to just explicitly copy the implementation and modify it a bit?

Reply:
It was trauma driven development after seeing what happens when all you have access to is a rendered Widget. 😅 

I will try to move past that though, and have deleted the builder method.

Reply:
Move over test-driven development, there's a new TDD in town